src/main/java/gov/nasa/jpl/mbee/mdk/ems/actions/CommitOrgAction.java

```diff
@@ -0,0 +1,148 @@
+    }
+
+
+    @Override
+    public void execute(Collection<Annotation> annos) {
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        commitAction();
+    }
+
+    public String commitAction() {
+        if (!MDUtils.isDeveloperMode()) {
+            return null;
+        }
+
+        // check for existing org
+        URIBuilder requestUri = MMSUtils.getServiceOrgsUri(project);
+        if (requestUri == null) {
+            return null;
+        }
+
+        JFrame selectionDialog = new JFrame();
+        String org = JOptionPane.showInputDialog(selectionDialog, "[DEVELOPER] Input MMS org below");
+        if (org == null || org.isEmpty()) {
+            Application.getInstance().getGUILog().log("[ERROR] Unable to commit org without name.");
+            return null;
+        }
+
+        ObjectNode response;
+        try {
+            response = MMSUtils.sendMMSRequest(MMSUtils.buildRequest(MMSUtils.HttpRequestType.GET, requestUri));
+            JsonNode arrayNode;
+            if ((arrayNode = response.get("elements")) != null && arrayNode.isArray()) {
+                for (JsonNode orgNode : arrayNode) {
+                    JsonNode value;
+                    if ((value = orgNode.get(MDKConstants.SYSML_ID_KEY)) != null && value.isTextual()) {
+                        if (value.asText() == org) {
+                            Application.getInstance().getGUILog().log("[WARNING] Org already exists. Creation will not proceed.");
+                            return org;
+                        }
+                    }
+                }
+            }
+        } catch (IOException | URISyntaxException | ServerException e1) {
+            Application.getInstance().getGUILog().log("[ERROR] Unable to query MMS orgs. Attempting to create without checking.");
+            e1.printStackTrace();
+        }
+
+        // build post data
+        ObjectNode requestData = JacksonUtils.getObjectMapper().createObjectNode();
+        ArrayNode elementsArrayNode = requestData.putArray("elements");
+        requestData.put("source", "magicdraw");
+        requestData.put("mdkVersion", MDKPlugin.VERSION);
+        ObjectNode orgObjectNode = JacksonUtils.getObjectMapper().createObjectNode();
+        orgObjectNode.put(MDKConstants.SYSML_ID_KEY, org);
+        orgObjectNode.put(MDKConstants.NAME_KEY, org);
+        elementsArrayNode.add(orgObjectNode);
+
+        // do post request
+        response = null;
+        try {
+            response = MMSUtils.sendMMSRequest(MMSUtils.buildRequest(MMSUtils.HttpRequestType.POST, requestUri, requestData));
+        } catch (IOException | URISyntaxException | ServerException e1) {
+            Application.getInstance().getGUILog().log("[ERROR] Unexpected error while committing org. Reason: " + e1.getMessage());
+            e1.printStackTrace();
+        }
+        JsonNode value;
+        if (response == null || (((value = response.get("message" )) != null) && value.isTextual()
```

Comment:
Not currently - failed creates return 200 and that message. It's still a bit of a moving target. Recommend tagging and coming back to later.